lms/validation/_api.py

```diff
@@ -33,6 +33,21 @@ class APIRecordSpeedgraderSchema(PyramidRequestSchema):
     """
 
 
+class APIReadResultSchema(PyramidRequestSchema):
+    """Schema for validating proxy requests to LTI Outcomes API for reading grades."""
+
+    locations = ["query"]
```

Comment:
I don't know that any of our other Schema classes specify `query` for `location`—this may be the only place where we're validating query params. Seems to work fine, though.

Reply:
Either `"querystring"` or `"querry"` should work fine: https://webargs.readthedocs.io/en/latest/quickstart.html#request-locations